packages/vchart/src/chart/util.ts

```diff
@@ -0,0 +1,21 @@
+import { Direction } from '../typings';
+import { merge } from '../util';
+import type { ICartesianChartSpec } from './cartesian/interface';
+
+export function setDefaultCrosshairForChart(spec: ICartesianChartSpec) {
+  spec.crosshair = merge(
+    {
+      [spec.direction === Direction.horizontal ? 'yField' : 'xField']: {
+        visible: true,
+        label: {
+          visible: false
+        },
+        line: {
+          visible: true,
+          type: 'rect'
+        }
+      }
+    },
+    spec.crosshair
```

Comment:
here, I think `spec.crosshair` might be an array.
![image](https://github.com/VisActor/VChart/assets/11360419/0f13b5bd-3b72-47b7-8e1e-8392f868affd)

Reply:
I have handled this, see https://github.com/VisActor/VChart/pull/83/commits/6ec9369b5c47fdb729f7d077bb0b3ae5bcda5a86